pypeit/scripts/flux_setup.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+import argparse
+import os,time
+import numpy as np
+from astropy.io import fits
+from astropy.table import Table
+from pypeit import msgs
+from pypeit.par.util import make_pypeit_file
+
+
+class SmartFormatter(argparse.HelpFormatter):
+
+    def _split_lines(self, text, width):
+        if text.startswith('R|'):
+            return text[2:].splitlines()
+        # this is the RawTextHelpFormatter._split_lines
+        return argparse.HelpFormatter._split_lines(self, text, width)
+
+def parser(options=None):
+    parser = argparse.ArgumentParser(description='Parse', formatter_class=SmartFormatter)
+    parser.add_argument("sci_path", type=str, help="Path for Science folder")
+    parser.add_argument("--objmodel", type=str, default='qso', choices=['qso', 'star', 'poly'],
+                        help="R|Science object model used in the telluric fitting.\n"
+                        "The options are:\n"
+                        "\n"
+                        "    qso  = For quasars. You might need to set redshift, bal_mask in the tell file.\n"
+                        "\n"
+                        "    star  = For stars. You need to set star_type, star_ra, star_dec, and star_mag in the tell_file.\n"
+                        "\n"
+                        "    poly = For other type object, You might need to set fit_region_mask, \n"
+                        "           and norder in the tell_file."
+                        )
+
+    if options is None:
+        args = parser.parse_args()
+    else:
+        args = parser.parse_args(options)
+    return args
+
+
+def main(args):
+    """
```

Comment:
Can you add a few more sentences on what this script does and what it will output. It sounds like it creates the coadd1d etc. files.

Reply:
done